crates/rooch/src/commands/init.rs

```diff
@@ -16,29 +19,86 @@ use rooch_types::crypto::RoochKeyPair;
 use rooch_types::error::RoochError;
 use rooch_types::error::RoochResult;
 use std::fs;
+use std::str::FromStr;
 
 /// Tool for init with rooch
 #[derive(Parser)]
 pub struct Init {
     /// Command line input of custom server URL
     #[clap(short = 's', long = "server-url")]
     pub server_url: Option<String>,
+    /// The address of the Rooch account to be set as key address
+    #[clap(short = 'k', long = "key-address")]
+    key_address: Option<String>,
```

Comment:
I think we do not need the `key_address` when `init`, the `key_address` is just a server start option.